### ISSUE TITLE
Allow Keyboard Commands to control playback

### ISSUE THREAD

**Issue by tehdef** [Musish/Musish#211]
I would like to be able to use my keyboard controls to instruct the music player to go back, go forward, pause and play. 

I am using Chrome.

I remember Google Play Music via Browser would listen to commands.

**Comment by ba5eem** [comment]
I was going to do a PR for this, but It has this feature already, unless I am misunderstanding what your asking.  Spacebar toggles play/pause; left=previous song & right=next song. I am also using Chrome. Am i missing something?

**Comment by tehdef** [comment]
Yeah. I have a keyboard that has the music controls in the top right corner. I was hoping to be able to use those to control it. Apps like Google Play Music can be controlled by those keys even when Chrome isn't being targeted. I could be playing another game and still control it. 

If you are curious what I'm talking about, I have a Razer Huntsman Elite. In the top right corner I have 4 buttons. 3 of those are for playback specifically.

**Comment by raphaelvigee** [comment]
Its not possible currently due to a browser limitation, feel free to reopen this with a solution. https://stackoverflow.com/a/9168544/3212099

**Comment by LofhJann** [comment]
Multimedia keys are natively supported since Chrome 73, play/pause works out-of-the-box. Not completely sure if next/previous buttons could be supported after as well.

**Comment by jordan-heemskerk** [comment]
@raphaelvigee Can we re-open this issue now that Chrome supports media keys? I'm happy to take a stab at an implementation too, although my web dev is rusty :P
